DQM/SiPixelMonitorDigi/interface/SiPixelDigiSource.h

```diff
@@ -21,26 +21,13 @@
 #include <memory>
 
 // user include files
-#include "FWCore/Framework/interface/Frameworkfwd.h"
-#include "FWCore/Framework/interface/EDAnalyzer.h"
-#include "FWCore/Framework/interface/MakerMacros.h"
-
-#include "DQMServices/Core/interface/DQMStore.h"
+#include "Geometry/TrackerGeometryBuilder/interface/TrackerGeometry.h"
+#include "Geometry/Records/interface/TrackerDigiGeometryRecord.h"
+#include "Geometry/CommonTopologies/interface/PixelTopology.h"
+#include "Geometry/Records/interface/TrackerTopologyRcd.h"
 #include "DQMServices/Core/interface/DQMOneEDAnalyzer.h"
-
 #include "DQM/SiPixelMonitorDigi/interface/SiPixelDigiModule.h"
 
-#include "DataFormats/Common/interface/DetSetVector.h"
-#include "DataFormats/SiPixelDigi/interface/PixelDigi.h"
-#include "DataFormats/SiPixelCluster/interface/SiPixelCluster.h"
-
-#include "FWCore/Framework/interface/EDProducer.h"
-#include "FWCore/Framework/interface/Event.h"
-#include "FWCore/Framework/interface/EventSetup.h"
-#include "DataFormats/Common/interface/Handle.h"
-#include "FWCore/Framework/interface/ESHandle.h"
-
-#include "FWCore/ParameterSet/interface/ParameterSet.h"
 #include <cstdint>
 
 class SiPixelDigiSource : public DQMOneLumiEDAnalyzer<> {
```

Comment:
I'm not sure what exactly needs to be done but I guess it could be incorporated in this PR (with an appropriate change in the PR title and description). Unless people prefer not to mix these two things.

Reply:
The problem is explained extensively in the thread of https://github.com/cms-sw/cmssw/issues/25090
There are some instructions on how to fix it here: https://indico.cern.ch/event/870369/contributions/3670615/attachments/2006693/3351542/dqm-one-and-legacy-modules.pdf

Reply:
@mmusich, I had a quick (probably too quick) look at the above links and couldn't quite figure out the extent of changes needed in the DQM code so in the interest of not delaying further this PR, I decided not to touch that part of the code.

Reply:
pity...

Reply:
Yes, it is a pity, this is one of the missing pieces, it would be nice to have a PR at some point